app/src/main/res/layout/list_facility_selection_progress.xml

```diff
@@ -1,42 +1,42 @@
 <?xml version="1.0" encoding="utf-8"?>
-<com.facebook.shimmer.ShimmerFrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto"
-  android:layout_width="match_parent"
-  android:layout_height="wrap_content"
-  app:shimmer_auto_start="true"
-  app:shimmer_base_color="#c8cbcc"
-  app:shimmer_colored="true"
-  app:shimmer_dropoff="1"
-  app:shimmer_duration="1200"
-  app:shimmer_highlight_color="@color/white"
-  app:shimmer_repeat_delay="400"
-  app:shimmer_tilt="15"
-  app:shimmer_width_ratio=".5">
+<androidx.cardview.widget.CardView style="@style/Clinic.FacilitySelection.CardView"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
 
-  <LinearLayout style="@style/Clinic.FacilitySelection.FacilityContainer">
+  <com.facebook.shimmer.ShimmerFrameLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:shimmer_auto_start="true"
+    app:shimmer_base_color="#c8cbcc"
```

Comment:
What is this color hex?
Is this the shimmer color?

Reply:
We made some changes to this layout: https://github.com/simpledotorg/simple-android/pull/523/commits/3713ec4ded91dff0447dc30c6b1cd99a1898ea53